bin/tictactoe

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+      
+Game.new.start  
```

Comment:
Great bin file, very short exactly as expected. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Separating the interface from the implementation of the game allows for multiple interfaces (e.g. a web app).